daprdocs/content/en/reference/api/configuration_api.md

```diff
@@ -0,0 +1,243 @@
+---
+type: docs
+title: "Configuration API reference"
+linkTitle: "Configuration API"
+description: "Detailed documentation on the configuration API"
+weight: 700
+---
+
+## Component file
+
+A Dapr `configuration.yaml` component file has the following structure:
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: <NAME>
+  namespace: <NAMESPACE>
+spec:
+  type: configuration.<TYPE>
+  version: v1
+  metadata:
+  - name:<KEY>
+    value:<VALUE>
+```
+
+| Setting | Description                                                                            |
+| ------- |----------------------------------------------------------------------------------------|
+| `metadata.name` | The name of the configuration store.                                                   |
+| `spec.metadata` | An open key value pair metadata that allows a binding to define connection properties. |
```

Comment:
This is a strange description
```suggestion
| `spec.metadata` | key/value pair metadata that is configured for each configuration store type. |
```

Reply:
This is not relevant anymore as this section was removed.